### PR TITLE
Adding 6 example dashboards with new colors and CSS

### PR DESCRIPTION
**Pull request by yousoph**
Updated colors, CSS, and positioning of charts in a few example dashboards: 
* Slack Dashboard
* COVID Vaccine Dashboard
* World Bank's Dashboard
* Sales Dashboard
* Video Game Sales
* FCC

### CODE REVIEW

**Review by rusackas — APPROVED** [review]
LGTM!

**Review by rusackas — APPROVED** [review]

File: charts/Cross_Channel_Relationship_heatmap_72.yaml [hunk 1]
```diff
@@ -51,7 +51,11 @@ params:
...
+  filter","time_range_endpoints":["inclusive","exclusive"],"url_params":{},"viz_type":"heatmap","xscale_interval":null,"y_axis_bounds":[null,null],"y_axis_format":"SMART_NUMBER","yscale_interval":null,"force":false,"result_format":"json","result_type":"full"},"result_format":"json","result_type":"full"}'
```
Comment (sadpandajoe) [inline charts/Cross_Channel_Relationship_heatmap_72.yaml:57]: should this be `heatmap` or `heatmap_v2`?

File: charts/Popular_Genres_Across_Platforms_39.yaml [hunk 2]
```diff
@@ -35,7 +35,11 @@ params:
...
+query_context: '{"datasource":{"id":2,"type":"table"},"force":false,"queries":[{"time_range":"No
+  filter","granularity":"year","filters":[],"extras":{"having":"","where":""},"applied_time_extras":{},"columns":[],"metrics":["count"],"annotation_layers":[],"row_limit":10000,"series_limit":0,"order_desc":true,"url_params":{},"custom_params":{},"custom_form_data":{}}],"form_data":{"adhoc_filters":[],"all_columns_x":"platform","all_columns_y":"genre","annotation_layers":[],"bottom_margin":"auto","canvas_image_rendering":"pixelated","datasource":"2__table","granularity_sqla":"year","left_margin":"auto","linear_color_scheme":"blue_white_yellow","metric":"count","normalize_across":"heatmap","queryFields":{"metric":"metrics"},"row_limit":10000,"show_legend":true,"show_perc":true,"show_values":true,"slice_id":99,"sort_x_axis":"alpha_asc","sort_y_axis":"alpha_asc","time_range":"No
+  filter","time_range_endpoints":["inclusive","exclusive"],"url_params":{},"viz_type":"heatmap","xscale_interval":null,"y_axis_bounds":[null,null],"y_axis_format":"SMART_NUMBER","yscale_interval":null,"force":false,"result_format":"json","result_type":"full"},"result_format":"json","result_type":"full"}'
```
Comment (sadpandajoe) [inline charts/Popular_Genres_Across_Platforms_39.yaml:41]: Same question here, should it be `heatmap_v2`?

File: charts/Vaccine_Candidates_per_Approach__Stage_15.yaml [hunk 3]
```diff
@@ -35,7 +35,11 @@ params:
...
+  annotation_layers: []
+query_context: '{"datasource":{"id":20,"type":"table"},"force":false,"queries":[{"time_range":"No
+  filter","filters":[],"extras":{"having":"","where":""},"applied_time_extras":{},"columns":[],"metrics":["count"],"annotation_layers":[],"row_limit":10000,"series_limit":0,"order_desc":true,"url_params":{},"custom_params":{},"custom_form_data":{}}],"form_data":{"adhoc_filters":[],"all_columns_x":"clinical_stage","all_columns_y":"product_category","annotation_layers":[],"bottom_margin":"auto","canvas_image_rendering":"pixelated","datasource":"20__table","left_margin":"auto","linear_color_scheme":"schemeYlOrBr","metric":"count","normalize_across":"heatmap","queryFields":{"metric":"metrics"},"row_limit":10000,"show_legend":false,"show_perc":true,"show_values":true,"slice_id":79,"sort_x_axis":"alpha_asc","sort_y_axis":"alpha_asc","time_range":"No
+  filter","time_range_endpoints":["inclusive","exclusive"],"url_params":{},"viz_type":"heatmap","xscale_interval":null,"y_axis_bounds":[null,null],"y_axis_format":"SMART_NUMBER","yscale_interval":null,"force":false,"result_format":"json","result_type":"full"},"result_format":"json","result_type":"full"}'
```
Comment (sadpandajoe) [inline charts/Vaccine_Candidates_per_Approach__Stage_15.yaml:41]: same here

File: charts/Vaccine_Candidates_per_Country__Stage_11.yaml [hunk 4]
```diff
@@ -2,29 +2,40 @@ slice_name: Vaccine Candidates per Country & Stage
 description: null
 certified_by: null
 certification_details: null
-viz_type: sunburst
+viz_type: heatmap
```
Comment (sadpandajoe) [inline charts/Vaccine_Candidates_per_Country__Stage_11.yaml:5]: Did we want this to be the old `heatmap` and not `heatmap_v2`

**Pull request merged**